README.md

```diff
@@ -110,13 +114,13 @@ You should be able to do the same in Mac or Windows but note that setting up a H
     Retype new password:
     passwd: all authentication tokens updated successfully.
     ```
-* As root edit /etc/sudoers and allow openbdre to perform `sudo`. Below will do it
+* As root edit `/etc/sudoers` and allow openbdre to perform `sudo`. Below will do it
```

Comment:
Probably don't need a code style highlight here as the code highlight is done immediately below